sql/mob_pools.sql

```diff
@@ -5953,131 +5953,131 @@ INSERT INTO `mob_pools` VALUES (5892,'Euryale','Euryale',171,0x06007106000000000
-INSERT INTO `mob_pools` VALUES (6015,'Mumor','mumor_ii',145,0x0000200C00000000000000000000000000000000,7,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,10,1,0,10);
-INSERT INTO `mob_pools` VALUES (6016,'Ingrid','ingrid_ii',145,0x00001E0C00000000000000000000000000000000,7,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,10,1,0,10);
-INSERT INTO `mob_pools` VALUES (6017,'Arciela','arciela_ii',145,0x00000D0C00000000000000000000000000000000,7,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,10,1,0,10);
-INSERT INTO `mob_pools` VALUES (6018,'Iroha','iroha_ii',145,0x0000280C00000000000000000000000000000000,7,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,10,1,0,10);
-INSERT INTO `mob_pools` VALUES (6019,'Shantotto','shantotto_ii',153,0x0000260C00000000000000000000000000000000,4,5,2,240,100,0,2,0,0,0,0,32,0,3,0,0,276,1,0,750);
+-- INSERT INTO `mob_pools` VALUES (6003,'cornelia','Cornelia',145,0x00002F0C00000000000000000000000000000000,21,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6004,'excenmille_s','Excenmille',145,0x0000EC0B00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6005,'ayame_uc','Ayame',149,0x0000F70B00000000000000000000000000000000,12,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6006,'maat_uc','Maat',149,0x0000F80B00000000000000000000000000000000,2,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6007,'aldo_uc','Aldo',149,0x0000F90B00000000000000000000000000000000,6,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6008,'naja_uc','NajaSalaheem',151,0x0000FA0B00000000000000000000000000000000,6,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6009,'lion_ii','Lion',149,0x0000090C00000000000000000000000000000000,6,13,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6010,'zeid_ii','Zeid',146,0x00000E0C00000000000000000000000000000000,8,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6011,'prishe_ii','Prishe',145,0x00000A0C00000000000000000000000000000000,2,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6012,'nashmeira_ii','Nashmeira',149,0x00000B0C00000000000000000000000000000000,18,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6013,'lilisette_ii','Lilisette',484,0x00000C0C00000000000000000000000000000000,19,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6014,'tenzen_ii','Tenzen',149,0x0000190C00000000000000000000000000000000,12,11,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6015,'mumor_ii','Mumor',149,0x0000200C00000000000000000000000000000000,4,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
+INSERT INTO `mob_pools` VALUES (6016,'ingrid_ii','Ingrid',149,0x00001E0C00000000000000000000000000000000,4,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,0,1,0,0);
```

Comment:
Ingrid II here is supposed to be WHM though.

Others: I have checked main/sub jobs for all other trusts as of this commit. You don't need to.